.changeset/twelve-shrimps-add.md

```diff
@@ -0,0 +1,9 @@
+---
+'@eth-optimism/contracts': patch
+'@eth-optimism/data-transport-layer': patch
+'@eth-optimism/integration-tests': patch
+'@eth-optimism/message-relayer': patch
+'@eth-optimism/sdk': patch
```

Comment:
```suggestion
'@eth-optimism/sdk': patch
'@eth-optimism/hardhat-node': patch
```

Reply:
Then can delete the other changeset file.

Reply:
I particularly had 2 changeset files because the auto generated changelogs create a link to the commit that the changeset was added in, so to use changesets correctly, you should have the changeset in the same commit as the changes it corresponds to